Makefile

```diff
@@ -0,0 +1,7 @@
+all:
```

Comment:
Instead of a Makefile would it be more obvious if this was just a script named `build-and-copy-isthmus-shadow-jar.sh`?

Reply:
Thanks for the suggestion, went with this approach.  The user is prompted for the absolute path of substrait-java now instead of needing to set the path first.